package dk.acto.fafnir.api.exception;

public class UserAlreadyExists extends RuntimeException {
}
